graphql_compiler/tests/schema_generation_tests/test_sqlalchemy_schema_generation.py

```diff
@@ -184,6 +186,80 @@ def test_index_generation_from_unique_constraint(self):
         )
 
 
+@pytest.mark.filterwarnings('ignore: Ignored * edges implied by composite foreign keys*')
```

Comment:
Are you sure this is correct? This seems to be written in glob syntax (`*` as a wildcard) rather than regex (`.*` as a wildcard). I don't remember which was expected by `filterwarnings` but I'd be somewhat surprised if it's glob syntax.